react/src/components/HubSection.tsx

```diff
@@ -198,8 +198,8 @@ export default function HubSection() {
                                         </p>
                                     </CardContent>
                                     <CardFooter className="p-4 pt-0 flex justify-center">
-                                        <Link to={`/package/${api.id}`}>
-                                            <Button variant="ghost" size="sm" className="text-blue-400 hover:text-blue-300 hover:bg-[#1e293b]">
+                                        <Link to={`/package/${api.id}`} tabIndex={0}>
+                                            <Button tabIndex={-1} variant="ghost" size="sm" className="text-blue-400 hover:text-blue-300 hover:bg-[#1e293b]">
```

Comment:
Wrapping a `Button` inside a `Link` like this can create accessibility issues. In HTML semantics, both `<a>` (used by `Link`) and `<button>` are interactive elements. Nesting them violates accessibility best practices and may confuse screen readers or disrupt keyboard navigation.

Reply:
From what I can see, the button is polymorphic, so we can put the link in the button this way:
```suggestion
                                        <Button variant="ghost" size="sm" className="text-blue-400 hover:text-blue-300 hover:bg-[#1e293b]" asChild>
                                          <Link to={`/package/${api.id}`} tabIndex={0}>  
```

Reply:
LGTM. Nice catch again @pplancq! @CroquetMickael : could you push another commit to fix both comments?